### ISSUE TITLE
[groups] allow any monitor in a group satisfy heartbeat interval

### ISSUE THREAD

**Issue by zombiehoffa** [louislam/uptime-kuma#4458]
### 📑 I have found these related issues/pull requests

I don't see any related issues in the search.

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

Right now  it appears as if a group takes its notification settings and applies it individually to all the monitors so that if any monitor fails the criteria it generates a notification for that child monitor. I would like the option to group monitors and allow any of them to satisfy the heartbeat condition of the group. I would use  this to setup a couple different methods per endpoint (i.e. push and ping) and group them togehter so that if either comes in in my time period it is considered up and if none come in it sends an alert.

### ✔️ Solution

add another advanced check box to group setup to treat all group monitors basically as one monitor.

**Comment by CommanderStorm** [comment]
Duplcate of #3387

TLDR:
- #4395
- a different PR enhancing the monior and implementing the different grouping modes
